iiasa/message_ix
# Add a module for implementing (sub-annual) time slices

This PR supersedes iiasa/message-ix-models#197 as @khaeru suggested this tool might be better placed in message_ix. 

@behnam-zakeri Please take a look at the changes. Several things come to mind before merging this: 

- [ ] Use the existing logging functionality instead of printing to stdout
- [ ] Write a bit of documentation on how to use this tool (e.g. via a `README.rst` as was done for `add_year`)
- [ ] Write tests for this new functionality.

## How to review

Check the above boxes plus:

- Read the diff and note that the CI checks all pass.
- Run a specific code snippet or command and check the output.
- Build the documentation and look at a certain page.
- Ensure that changes/additions are self-documenting, i.e. that another
  developer (someone like the reviewer) will be able to understand what the code
  does in the future.

## PR checklist
